src/main/java/uk/ac/ebi/ampt2d/accession/server/ApiError.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.server;
+
+import org.springframework.http.HttpStatus;
+
+public class ApiError {
+
+    private HttpStatus status;
+    private String message;
+    private String debugMessage;
+
+    private ApiError() {
```

Comment:
Not needed, constructors should call the most generic one and define which values are gonna initialize the private elements.

On the same note, favour asignation at construction, do not use setters if it is not required, knowing the use of this class all the parameters should be initializated on creation and also be marked as final.